interface/src/app/node/high_range_exp.tpl.html

```diff
@@ -7,7 +7,7 @@
   <button class="btn btn-primary" ng-click="setMode()"
           ng-show="topMode">Show All</button>
 
-  <br />
+  <br>
```

Comment:
Nice catch. We are serving `text/html`, not `application/xhtml+xml`, so this change is good.